types.go

```diff
@@ -258,6 +260,109 @@ type VirtualMachineConfig struct {
 	Boot    string
 	VMGenID string
 	Name    string
+
+	Ides map[string]string
+	Ide0 string
+	Ide1 string
+	Ide3 string
+	Ide4 string
+	Ide5 string
+	Ide6 string
+	Ide7 string
+	Ide8 string
+	Ide9 string
+
+	Scsis map[string]string
+	Scsi0 string
```

Comment:
there is a `SCSI0` at ln 259, I was trying to abide by the acronym is all caps thing that go linters try to enforce. you ok refactoring and use SCSI0-9, IDE0-9 etc?

Reply:
changed